test/unit/wiki_extensions_child_pages_count_test.rb

```diff
@@ -4,19 +4,40 @@ class WikiExtensionsChildPagesCountTest < Redmine::HelperTest
   include ApplicationHelper
   include ERB::Util
 
-  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues
+  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues,
+           :roles, :enabled_modules, :users
+
+  def setup
+    super
+    @project = nil
+  end
 
   def test_child_pages_count_in_wiki_content
```

Comment:
`WikiExtensionsChildPagesCountTest`という名前の中なので`child_pages_count_`は冗長です。

```suggestion
  def test_wiki_content
```

Reply:
[7f22c70](https://github.com/otegami/redmine_wiki_extensions/pull/5/commits/7f22c70b31611378cefdf1935b0dbb7c2e55f723) メソッド名が確かに冗長だったので修正します。